html-aam/accname-computation-by-element/inputs-with-placeholders.html

```diff
@@ -0,0 +1,47 @@
+<!doctype html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <title>Accname computation tests for input elements that support a placeholder attribute</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/resources/testdriver.js"></script>
+  <script src="/resources/testdriver-vendor.js"></script>
+  <script src="/resources/testdriver-actions.js"></script>
+  <script src="/wai-aria/scripts/aria-utils.js"></script>
+  <script src="/html-aam/scripts/html-aam-accname-utils.js"></script>
+</head>
+
+<body>
+  <script>
+    const testConfig = {
+      urlSpec: 'https://www.w3.org/TR/html-aam-1.0/#input-type-text-input-type-password-input-type-number-input-type-search-input-type-tel-input-type-email-input-type-url-and-textarea-element-accessible-name-computation',
+      elements: {
+        'input[type="email"]': { attrs: ['value'] },
+        'input[type="number"]': { attrs: ['value'] },
+        'input[type="password"]': { attrs: ['value'] },
+        'input[type="search"]': { attrs: ['value'] },
+        'input[type="tel"]': { attrs: ['value'] },
+        'input[type="text"]': { attrs: ['value'] },
+        'input[type="url"]': { attrs: ['value'] },
+        'textarea': { descendants: ['textNode'] }
+      },
+      nameSources: [
+        'from 2 aria-labelledby refs',
+        'from 1 aria-labelledby ref',
+        'from aria-label',
+        'from 2 labels',
+        'from 1 label',
+        'from title',
+        'from placeholder',
```

Comment:
It's not in the specs, but should we consider adding aria-placeholder between title and placeholder?

Reply:
seems the reality is that the native attribute takes precedent over the aria-placeholder attribute.

firefox on macOS is the only one i found that behaves a bit different, and VoiceOver actually would combine the values of `placeholder=n` and `aria-placeholder=p` to be announced as "n p"

going to make an HTML AAM update for this to resolve the "it's not in the specs" part of your comment :)